code/modules/lore_hardcoded/faction.dm

```diff
@@ -103,48 +79,40 @@
 	government or Corporate space. The FTU are the originators of the Tradeband language, created specially to serve as the lingua franca for Merchants \
 	across the Galaxy, to ensure members may understand each other regardless of native language or nationality."
 	contractor_info = "Working under a Corporation feels strange. You're used to exercising your autonomy, or perhaps you're used to the more strict maritime environment of your former vessel. Being surrounded by Corporate employees reminds you of why you're with the FTU."
+	// FTU is not a trusted / allied faction.
+	// They get a whitelist, not a blacklist.
 	job_whitelist = list(
```

Comment:
I noticed the FTU lacks the trader job. Wouldn't this lock them out of being the main faction that is traders to begin with?